openvdb/openvdb/python/README.md

```diff
@@ -0,0 +1,87 @@
+# `openvdb`
+
+This project contains python bindings for OpenVDB.
+
+## Dependencies
+
+| Name    | Version |
+| ---     | ---     |
+| cmake   | >=3.15  |
+| numpy   | >=1.14  |
+| openvdb | >=9.0.0 |
+
+## Installing from source
+
+Installation using `pip` is preferred because it manages the package version
+sensibly, and also automatically handles finding the correct python
+installation.
+
+### Using pip
+
+The python bindings for OpenVDB can be installed using `pip install .`. In this
+case, the build build toolchain executes in the following order:
+
+1. The project's `pyproject.toml` is read, invoking the build backend. The
+   `openvdb` python bindings use `scikit-build-core` as a build backend.
+   `scikit-build` acts as [a wrapper for `cmake`][scikit-build].
```

Comment:
`scikit-build` isn't used not required from what I see in this PR.

Reply:
Nice catch, changing now.